tests/filesTest.js

```diff
@@ -1082,8 +1085,7 @@ describe('Main: Currently testing files management,', function () {
           }
         })
         .willRespondWith({
-          status: 201,
-          headers: htmlResponseHeaders
```

Comment:
NOTE: a COPY request's reponse doesn't have content-type header in ocis server as oc10 does.
As client doesn't need `content-type` here, I feel safe to remove this line.

or any thoughts?

Reply:
Yes , I removed the `headers` part , only `status` is present.